device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/edge/DirectMethodResponse.java

```diff
@@ -0,0 +1,74 @@
+/*
+ *  Copyright (c) Microsoft. All rights reserved.
+ *  Licensed under the MIT license. See LICENSE file in the project root for full license information.
+ */
+
+package com.microsoft.azure.sdk.iot.device.edge;
+
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonElement;
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor
+public class DirectMethodResponse
+{
+    private static final String STATUS_KEY_NAME = "status";
+    @Expose(serialize = false)
+    @SerializedName(STATUS_KEY_NAME)
+    private int status;
+
+    private static final String PAYLOAD_KEY_NAME = "payload";
+    @Expose(serialize = false)
+    @SerializedName(PAYLOAD_KEY_NAME)
+    private JsonElement payload;
+
+    //empty constructor for gson
+    private DirectMethodResponse()
+    {
+
+    }
+
+    public DirectMethodResponse(String json)
+    {
+        DirectMethodResponse result = new GsonBuilder().create().fromJson(json, DirectMethodResponse.class);
+
+        this.payload = result.payload;
+        this.status = result.status;
+    }
+
+    public int getStatus()
```

Comment:
You can annotate the private field ```status``` with the lombok annotation ```@Getter``` to make this getter method for you